static/apidocs/zoom.js

```diff
@@ -0,0 +1,9 @@
+import mediumZoom from 'medium-zoom'
```

Comment:
For this to work, u need to run a `npm install medium-zoom` first, that explains why it shows a 404.